internal/pkg/service/common/etcdclient/etcdclient.go

```diff
@@ -67,12 +69,19 @@ func New(ctx context.Context, proc *servicectx.Process, tel telemetry.Telemetry,
 	// Create connect context
 	connectCtx, connectCancel := context.WithTimeout(ctx, cfg.ConnectTimeout)
 	defer connectCancel()
+	connectCtx = ctxattr.ContextWith(
+		connectCtx,
+		attribute.String("etcd.connect.timeout", cfg.ConnectTimeout.String()),
+		attribute.String("etcd.keepAlive.timeout", cfg.KeepAliveTimeout.String()),
+		attribute.String("etcd.keepAlive.interval", cfg.KeepAliveInterval.String()),
+		attribute.StringSlice("etcd.endpoints", []string{cfg.Endpoint}),
+	)
```

Comment:
Cannot this be injected into new `WithoutCancel` context?

Reply:
I don't understand.

Reply:
https://github.com/keboola/keboola-as-code/pull/1852/files/9f96e7cff95810f8057dc2a7ada622fd6f507d75#diff-c072bc78b2cf15de529a784c1ebaf3ac7258572159b8aaec76bdeba4d19ca6f3R84
Here you create `context.WithoutCancel(ctx)`. Cannot this context has this attributes?

Reply:
No, I want this info only in the connect ctx (so in the connect logs), not in all etcd client logs.

Reply:
`context.WithoutCancel` is long-living context for the ectd client - no only for the connect operation